protostar/cheatable_starknet/controllers/contracts.py

```diff
@@ -113,10 +170,51 @@ async def _transform_calldata_to_cairo_data(
                 ) from dt_exc
         return calldata or []
 
+    async def declare_sierra_contract(
+        self,
+        contract_class: ContractClass,
+        compiled_class: CompiledClass,
+    ) -> DeclaredSierraClass:
+        """
+        Declare a sierra contract.
+
+        @param contract_class: sierra compiled contract to be declared
+        @param compiled_class_hash: compiled class hash of the casm compiled contract.
+        @param compiled_class: casm compiled contract to be declared
```

Comment:
Why we have the same `@param` repeated twice, it is some python docsting convention for alternative?

Reply:
Just my error